File: Python/tests/test_choiceRT_ddm.py
import pytest

from hbayesdm.models import choiceRT_ddm


def test_choiceRT_ddm():
    _ = choiceRT_ddm(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_choiceRT_ddm.R
context("Test choiceRT_ddm")
library(hBayesDM)

test_that("Test choiceRT_ddm", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(choiceRT_ddm(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
